subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/IncrementalScalaCompileIntegrationTest.groovy

```diff
@@ -36,12 +35,8 @@ class IncrementalScalaCompileIntegrationTest extends AbstractIntegrationSpec {
             compileScala.options.debug = false
 '''
         then:
-        // This gets reset each time you run() is run.
-        executer.expectDeprecationWarning();
```

Comment:
Thanks. I've added this PR to clean up redundant deprecation warning checks: https://github.com/gradle/gradle/pull/2448